### ISSUE TITLE
[BUG] Web: Cannot scroll sub-events for super or recurring events

### ISSUE THREAD

**Issue by vburgett** [rokwire/illinois-app#4999]
When using the website, users can not scroll through sub-events for super- or recurring events unless they have a track pad. 

Do you need design ideas to fix this? 

![Image](https://github.com/user-attachments/assets/e254aa7f-3a99-43ee-9d61-3b9a5e95738e)
